darts/tests/test_probabilistic_models.py

```diff
@@ -77,18 +98,67 @@ def helper_test_probabilistic_forecast_accuracy(self, model_cls, model_kwargs, e
         mae_err_median = mae(ts[100:], pred)
         self.assertLess(mae_err_median, err)
 
-        # test accuracy for increasing quantiles between 0.7 and 1 (it should decrease)
+        # test accuracy for increasing quantiles between 0.7 and 1 (it should ~decrease)
         tested_quantiles = [0.7, 0.8, 0.9, 0.99]
         mae_err = mae_err_median
         for quantile in tested_quantiles:
             new_mae = mae(ts[100:], pred.quantile_timeseries(quantile=quantile))
-            self.assertLess(mae_err, new_mae)
+            self.assertLess(mae_err, new_mae+0.1)
             mae_err = new_mae
 
-        # test accuracy for decreasing quantiles between 0.3 and 0 (it should decrease)
+        # test accuracy for decreasing quantiles between 0.3 and 0 (it should ~decrease)
         tested_quantiles = [0.3, 0.2, 0.1, 0.01]
         mae_err = mae_err_median
         for quantile in tested_quantiles:
             new_mae = mae(ts[100:], pred.quantile_timeseries(quantile=quantile))
-            self.assertLess(mae_err, new_mae)
+            self.assertLess(mae_err, new_mae+0.1)
             mae_err = new_mae
+
+    """ More likelihood tests
```

Comment:
@pennfranc I've added unit tests for all likelihoods here. Could you maybe take a quick look? Thx :)

Reply:
Looks good!